tracy/cs/homepage.texy

```diff
@@ -288,6 +288,18 @@ Výsledek vypadá asi takto:
 [* tracy-firelogger.png *]
 
 
+Portů
+-----
+
+Toto je seznam neoficiálních portů pro další rámce a CMS:
```

Comment:
Your czech is really good :-))

It should be:

Porty
— —

Toto je seznam neoficiálních portů pro další frameworky a CMS: